kinesis-s3/stop.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Source library 
+. ../utils/helper.sh
+
+source config/demo.cfg
+SCHEMA_REGISTRY_CONFIG_FILE=$HOME/.ccloud/config
+#SCHEMA_REGISTRY_CONFIG_FILE=schema_registry.config
+../ccloud/ccloud-generate-cp-configs.sh $SCHEMA_REGISTRY_CONFIG_FILE
+source delta_configs/env.delta
+
+check_env || exit 1
+
+aws kinesis delete-stream --stream-name $KINESIS_STREAM_NAME
+aws s3 rm --recursive s3://$DEMO_BUCKET_NAME/topics/${KAFKA_TOPIC_NAME_OUT}
+aws s3 rm --recursive s3://$DEMO_BUCKET_NAME/topics/COUNT_PER_CITY
+rm -f data.avro
+
+# Delete topics in Confluent Cloud
+topics=$(ccloud topic list)
+topics_to_delete="$KAFKA_TOPIC_NAME_IN $KAFKA_TOPIC_NAME_OUT COUNT_PER_CITY connect-configs connect-statuses connect-offsets"
```

Comment:
Should `LOCATIONS` be in this list of topics to delete?

Reply:
There is no `LOCATIONS` topic to delete.  The stream of that name has the demo topic name:

```bash
ksql> show streams;

 Stream Name | Kafka Topic       | Format    
---------------------------------------------
 LOCATIONS   | kinesis-s3-topic1 | DELIMITED 
---------------------------------------------
```